### PR TITLE
Fix crlf eol

### PR DESCRIPTION
**Pull request by sof202**
# Description

This pull request will fix crlf issues

## Issue ticket number

This pull request is to address issue: #43 .

## Type of pull request

- [x] Bug fix
- [ ] New feature/enhancement
- [ ] Code refactor
- [ ] Documentation update

## Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested my code to check that it is functional
- [ ] I have used linters to check for common sources of errors
- [ ] I have implemented fail safes in my code to account for edge cases
- [ ] I have made the corresponding changes to the documentation

### CODE REVIEW

**Review by eld211 — COMMENTED** [review]
Looks ok to me :-)

**Review by SiyiSEA — DISMISSED** [review]
good

File: sequencing/DNAhydroxy/jobSubmission/peakCalling5hmCIncreasingSampleSizes.sh
Comment (ejh243) [inline sequencing/DNAhydroxy/jobSubmission/peakCalling5hmCIncreasingSampleSizes.sh:1]: Not sure you meant to commit this.

**Review by ejh243 — CHANGES_REQUESTED** [review]
sequencing/DNAhydroxy/jobSubmission/peakCalling5hmCIncreasingSampleSizes.sh needs attention. 
Otherwise fine for merge.

**Review by ejh243 — CHANGES_REQUESTED** [review]
Some funny formatting going on here....

**Comment by sof202** [comment]
@ejh243 Fine, I fixed it.

File: sequencing/DNAhydroxy/jobSubmission/peakCalling5hmCIncreasingSampleSizes.sh [hunk 1]
```diff
@@ -1,61 +1,63 @@
...
-PULLDOWNS=($(ls *PC*.GRCh38.karyo.deduplicated.bam))
-
-for NSAMPLES in {2..19}
-do
-	macs2 callpeak -t ${PULLDOWNS[@]:0:${NSAMPLES}} -c ${CONTROLS[@]:0:${NSAMPLES}} -f BAM -g hs -n All5hmCPeakCalling_${NSAMPLES}_Samples -B -q 0.01 --outdir ${PEAKDIR}
-done
-
-
+
#!/bin/sh
```
Comment (ejh243) [inline sequencing/DNAhydroxy/jobSubmission/peakCalling5hmCIncreasingSampleSizes.sh:1]: I have got a weird character at the beginning of this line

**Review by ejh243 — APPROVED** [review]

File: sequencing/DNAhydroxy/jobSubmission/peakCalling5hmCIncreasingSampleSizes.sh (hunk 1 above)
Comment (sof202) [reply sequencing/DNAhydroxy/jobSubmission/peakCalling5hmCIncreasingSampleSizes.sh:1]: yeah, alright you got me.

**Pull request merged**